src/praisonai-agents/praisonaiagents/tools/jira_tools.py

```
@@ -0,0 +1,377 @@
+            result += f"Description: {issue.fields.description[:500]}...\n"
+        
+        # Recent comments
+        comments = jira.comments(issue_key)
+        if comments:
+            result += f"\nRecent Comments ({len(comments[-3:])}):\n"
+            for comment in comments[-3:]:
+                result += f"  - {comment.author.displayName} ({comment.created}): {comment.body[:200]}...\n"
+        
+        return result
+        
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)